skunkworks_crow/src/main/res/drawable/ic_security_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
If you've used any tool to generate these icons then please do mention about those, its always good practice to mention someone else's work if we're building over that.

Reply:
Sure! These icons are generated by Android Studio, so I think we don't need to do that. But I'll mention other icons get from internet.